### DISCUSSION TITLE
withdraw

### DISCUSSION THREAD

**Discussion by kafraxxx** [blockscout/blockscout#9149] [Support]
hey guys i started withdraw that i want to cancel. Is it possible, my funds are lost because i didnt prove and i won t it, i just try to withdraw to test but i want cancel it .
![Capture d'écran 2024-01-13 120008](https://github.com/blockscout/blockscout/assets/152737840/fc8bfece-7bcc-4dd7-9695-34af0beb09c0)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer, so we do not host this chain ourselves. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
